nexus-tweaks.user.js

```diff
@@ -1144,6 +1144,264 @@ promiseList.push((async () => {
+            "Locks Picked": ["Pilferer", "Midnighter"],
+            "Pets Killed": ["Big Game Hunter", "Taxidermist"],
+            "Pills Taken": ["Junkie", "Cabinet Raider"],
+            "Power Removed": ["Pitchman", "Black Knight"],
+            "Power Restored": ["Linesman", "Head Linesman"],
+            "Targets Shot": ["Crack Shot", "Sniper"]
+        },
+        "Exploration" : {
+            "Breath 4" : [
+                "A New Chapter", "Academic Probation", "All In The Family", "And I Must Scream", "At All Costs", "Baraas Ascends", "Birthing Pool", "Broken Alliance",
+                "Broken Promises", "Circumnavigation", "Citadel", "Clinging to Life", "Cloudwatching", "Cops and Robbers", "Dedicated Few", "Enthroned", "Explosive Yield",
+                "Fall of the Watcher", "Four Corners", "Fragmented Return", "Halls of the Scholar", "Halls of Wrath", "Idle Hands", "In The Name Of Science", "Institute of Arts",
+                "Into the Dark", "Last Confession", "Reasons to Live", "Remorse", "Stolen Victory", "Tapestry of Time", "The Earth Shudders", "The Legend", "The Little King",
+                "The Rise of Kafa-El", "The Voice", "Under The Boot", "Untouched Wilderness", "Well of Truth", "What Once Was Lost"
+            ],
+            "Breath 3" : [
+                "Bloodlines", "Common Touch", "Cromahl-Hult", "Deluge", "Effervescence", "Eresius Rest", "Fatigued", "Favored of Baraas", "Fire and Ice", "Flakes of Delight",
+                "Fought the Law", "Four Winds", "Harmonious Joining", "Hole in None", "Immutable Law", "Landfall", "Netherland", "Paradise Lost", "Riptide", "Shades of Black",
+                "Shades of Grey", "Sincerest Flattery", "Tainted Fruit", "The Crypt of Maeval", "View from the Top"
+            ],
+            "Breath 2" : [
+                "A Whole New World", "All Ye All Ye Outs In Free", "Beginning of the End", "Blinded by Lust", "Bloodlust", "Bridge Magic Built", "Captain Bloodbeards Folly",
+                "Crystal Farm", "Deck of Cards", "Goldlust", "Ground Zero", "Impassable Dream", "Last Ship Leaving", "Mene Mene Tekel Upharsin", "Smiles All Around", "Snicker-Snack",
+                "Time Sink", "Tree Hugger", "Viva La Revolucion", "Walking Home", "Watery Grave", "When The Walls Came Tumbling Down"
+            ],
+            "Breath 1" : [
+                "Altar of the Ancients", "Big Bang", "Confluence", "Founding Father", "Great Bowl", "Hermits Repose", "Iron Barracks", "Lord Mayor", "Shrine of the Wound", "Slowpoke"
+            ]
+        },
+        "Monster Hunting" : {
+            "Crystal Guardian" : ["Shardbreaker", "Crystal Smasher", "Entropy Incarnate"],
+            "Great Wyrm" : ["Wyrm Hunter", "Wyrm Slayer", "Wyrmsbane"],
+            "Maeval" : ["Stalker of Death", "Hunter of Death", "Foe of Death"]
+        }
+    }
+
+    function classifyBadges(badgesText) {
+        const badgeTexts = badgesText.trim().split(",").map(str => str.trim())
```

Comment:
Use single quotes for consistency, except where there's single quotes in the string and except in the badge names (for consistency since some have apostrophes).

https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes